deno/payloads/v10/_interactions/_applicationCommands/_chatInput/string.ts

```diff
@@ -8,11 +8,11 @@ import type { APIApplicationCommandOptionChoice, ApplicationCommandOptionType }
 interface APIApplicationCommandStringOptionBase
 	extends APIApplicationCommandOptionBase<ApplicationCommandOptionType.String> {
 	/**
-	 * For option type `STRING`, the minimum allowed length (minimum of 0).
+	 * For option type `STRING`, the minimum allowed length (minimum of 0, maximum of `6000`).
```

Comment:
```suggestion
	 * For option type `STRING`, the minimum allowed length (minimum of `0`, maximum of `6000`).
```
Since you are using in the maximum. Or just delete these